tests/app/celery/test_ftp_update_tasks.py

```diff
@@ -39,9 +39,11 @@
 @pytest.fixture
 def notification_update():
     """
-    Returns a namedtuple to use as the argument for the check_billable_units function
+    Returns an instance of the  NotificationUpdate dataclass to use as the argument
+    for the check_billable_units function
     """
-    NotificationUpdate = namedtuple("NotificationUpdate", ["reference", "status", "page_count", "cost_threshold"])
+    from app.celery.tasks import NotificationUpdate
```

Comment:
these inline imports are probably a mild hint that this class isn't defined in a great place

I wonder if we should move it to a more central file like `dataclasses.py` (sibling to our `models.py`) so that these can be imported/used from anywhere without worrying about circular imports.